src/main/java/com/ceph/rados/Rados.java

```diff
@@ -402,4 +402,122 @@ public void shutDown() {
             this.clusterPtr = null;
         }
     }
+
+    /**
+     * Executes commands in module: 'mon'
+     *
+     * To list commands execute "get_command_descriptions"
+     *
+     * @param commands Commands to execute
+     * @param in inputbuffer
+     * @return Returns a RadosCommandResult-object
+     * @throws RadosException
+     */
+    public RadosCommandResult executeRadosMonCommand(final String[] commands, final String in) throws RadosException {
+        final PointerByReference outBuf = new PointerByReference();
+        final IntByReference outBufLen = new IntByReference();
+        final PointerByReference statusBuf = new PointerByReference();
+        final IntByReference statusBufLen = new IntByReference();
+
+        int r = rados.rados_mon_command(this.clusterPtr, commands, commands.length, in, in.length(), outBuf, outBufLen, statusBuf, statusBufLen);
+
+        final String status = (statusBuf.getValue() == null ? "" : new String(statusBuf.getValue().getByteArray(0, statusBufLen.getValue())));
+
+        if (r != 0)
+            throw new RadosException("Error when executing Rados monitor commands:\n " + status, r);
+
```

Comment:
Just force of habit, enhances readability I guess, if you wish I can remove those